src/components/Heading.js

```diff
@@ -44,9 +44,9 @@ function Heading() {
 
       <div className="image-row">
         <div className="apply-content">
-          <a href="#Apply"><img src={apply_button}></img></a>
+          <a href="https://apply.pennapps.com/" target="_blank"><img className="animated-button" src={apply_button}></img></a>
```

Comment:
Could you revert this change? We do want the button to link to the Apply section so that they read that section before applying

Reply:
Actually let's wait on this; creative is changing decisions

Reply:
Okay let's keep this change as creative requested